### PR TITLE
Deprecate minzoom and maxzoom in favor of minimum-scale-denominator and maximum-scale-denominator

### PR DESCRIPTION
**Pull request by talaj**
It was done by https://github.com/mapnik/mapnik/commit/219ad1f2ccd2eaa5799dad73236378b127f490d9 in Mapnik.

### CODE REVIEW

File: 3.0.0/reference.json [hunk 1]
```diff
@@ -138,13 +138,27 @@
             "default-value": 0,
             "type":"float",
             "default-meaning": "The layer will be visible at the minimum possible scale",
-            "doc": "The minimum scale denominator that this layer will be visible at. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6."
+            "doc": "The minimum scale denominator that this layer will be visible at. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6. This option has been deprecated in favor of `minimum-scale-denominator`.",
+            "status": "deprecated"
```
Comment (yohanboniface) [inline 3.0.0/reference.json:142]: Was it deprecated in `3.0.0` yet?

File: 3.0.0/reference.json [hunk 2]
```diff
@@ -138,13 +138,27 @@
...
         },
         "maxzoom": {
             "default-value": "1.79769e+308",
             "type":"float",
             "default-meaning": "The layer will be visible at the maximum possible scale",
-            "doc": "The maximum scale denominator that this layer will be visible at. The default is the numeric limit of the C++ double type, which may vary slightly by system, but is likely a massive number like 1.79769e+308 and ensures that this layer will always be visible unless the value is reduced. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6."
+            "doc": "The maximum scale denominator that this layer will be visible at. The default is the numeric limit of the C++ double type, which may vary slightly by system, but is likely a massive number like 1.79769e+308 and ensures that this layer will always be visible unless the value is reduced. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6. This option has been deprecated in favor of `maximum-scale-denominator`.",
```
Comment (yohanboniface) [inline 3.0.0/reference.json:148]: For my understanding, and in two words, why `-1e-6` and `+1e-6`? :)

File: 3.0.0/reference.json (hunk 1 above)
Comment (talaj) [reply 3.0.0/reference.json:142]: Take a look on https://github.com/mapnik/mapnik/commit/219ad1f2ccd2eaa5799dad73236378b127f490d9, there are all git tags including all 3.0.x versions.

Comment (yohanboniface) [reply 3.0.0/reference.json:142]: Thanks :)

**Review by yohanboniface — APPROVED** [review]

File: 3.0.0/reference.json (hunk 2 above)
Comment (talaj) [reply 3.0.0/reference.json:148]: This is a kind of epsilon used in Mapnik [here](https://github.com/mapnik/mapnik/blob/b4bce33b34f767883e8df0ff96719b59f3f91995/src/layer.cpp#L215).

**Pull request merged**